src/pages/preferences/components/Row.vue

```diff
@@ -0,0 +1,56 @@
+<template>
```

Comment:
Do we need this row component? I would rather have it into template fully. 

Would maybe fine if component is layout only, but this one includes logic.